roles/st2/tasks/redhat.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Install latest RHEL st2 package (stable)
+  become: yes
+  yum:
+    name: st2
+    state: latest
+    disable_gpg_check: yes
```

Comment:
Why we have `disable_gpg_check: yes` for `stable` and nothing for `unstable` in next block? Any specific reason?